spec.bs

```diff
@@ -2495,6 +2495,23 @@ container/fenced navigable=] will fail, as outlined in [[#navigation-patch]].
 
 </div>
 
+<h4 id=allow-automatic-beacons>The
+\`<a http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></a>\` HTTP response header</h4>
+
+A {{Document}} that is not [=same origin=] with its [=Document/browsing context=]'s [=fenced frame
+config instance=]'s [=fenced frame config instance/mapped url=] can opt into sending an automatic
+beacon when it navigates by using the <dfn
+http-header><code>Allow-Fenced-Frame-Automatic-Beacons</code></dfn> HTTP response header. This
+header is a [=structured header=] whose value must be either the [=string=] "true" or "false".
+
+<h4 id=allow-cross-origin-reporting>The
+\`<a http-header><code>Allow-Cross-Origin-Event-Reporting</code></a>\` HTTP response header</h4>
+
+A {{Document}} loaded from a [=fenced frame config instance=] can opt into its cross-origin [=child
+navigables=]' {{Document}}s sending {{Fence/reportEvent()}} beacons by using the <dfn
+http-header><code>Allow-Cross-Origin-Event-Reporting</code></dfn> HTTP response header. This header
```

Comment:
Documents can't use the header, but the server can set them. Or do you mean the document can literally use the header somehow with a web platform API? I would probably say something like: serving the document resource that gets loaded into a `<fencedframe>` by a fenced frame config instance with a FOO header can automatically opt in the document and its cross-origin child navigables to be able to send reportEvent() beacons etc. I think this is more clear since it traces the chronology / flow of how it actually works.